src/docs/content/docs/usage/sidecar_mode.md

```diff
@@ -0,0 +1,49 @@
+
+
+## Guidance
+
+ * *Deploy one Reaper cluster per Cassandra cluster*: Sidecar mode has been designed to allow you to easily deploy a separate, highly available Reaper service for each of your Cassandra clusters.
+   You can use a Sidecar Mode Reaper service to manage multiple clusters, but this is not the use-case for which it has been designed and tested.
+ * *Deploy Reaper in a sidecar container alongside Cassandra*: If you are using Kubernetes to deploy Cassandra and Reaper, put the Reaper process in the same Pod as Cassandra, in a sidecar container. See [Pods that run multiple containers that need to work together](https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/)
+ * *Use Spreaper and the REST API for Reaper administration (use the WebUI for reporting)*: If you have automated the deployment of Reeaper in Sidecar Mode alongside a Cassandra Cluster (described above), you can also [Preconfigure that cluster](https://github.com/thelastpickle/cassandra-reaper/pull/425) by writing a script that interacts with the Reaper REST API (or using spreaper) to automatically configure the repair settings for that cluster.
+ * *Use or write an operator*: For example, you could write a [Kubernetes Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) (e.g. a `Cassandra` resource) which contains configuration fields for the Cassandra database and fields with which you can define the Reaper repair settings.
+   Then write an accompanying [Kubernetes Operator](https://coreos.com/operators/) which watches for changes to those resources and reconciles the declared (desired) configuration with the actual state of the Cassandra cluster and its Reaper service.
+
+## Caveats
+
+ * *Cassandra Backend Only*: When operating in this mode, Reaper must be configured to use the [Cassandra Backend](../../backends/cassandra).
+ * *One Reaper cluster per Cassandra Cluster*: Sidecar Mode is designed to be used in situations where you deploy a separate Reaper cluster for every Cassandra cluster.
+   If you prefer to deploy a single Reaper cluster to manage multiple Cassandra clusters then Sidecar Mode may not be suitable for you.
+ * *Container orchestration (Kubernetes) preferred*: Following on from the "one-reaper-per-cluster" caveat (above) it is also worth noting that it is easiest to deploy Reaper in Sidecar Mode if your Cassandra cluster is managed by a container orchestration system, such as Kubernetes.
+   Kubernetes ensures that the Reaper process starts and dies with the Cassandra process.
+   Kubernetes also ensures that the Reaper process shares a network namespace with the Cassandra process so that it (and only it) can access the JMX service.
+   If you are not using Kubernetes (or similar) it may be overly complicated to deploy in a secure and resilient Sidecar Mode Reaper cluster.
```

Comment:
Again, I disagree.
What makes you think it would be overly complicated in a non containerized environment?